editor/resources/editor/css/initial-load.css

```diff
@@ -1,62 +1,5 @@
-@font-face {
-  font-family: 'utopian-inter';
-  src: url('../fonts/Inter-Regular.woff2') format('woff');
-  font-weight: 400;
-}
-
-@font-face {
-  font-family: 'utopian-inter';
-  src: url('../fonts/Inter-Italic.woff2') format('woff');
-  font-weight: 400;
-  font-style: italic;
-}
-
-@font-face {
-  font-family: 'utopian-inter';
-  src: url('../fonts/Inter-Medium.woff2') format('woff');
-  font-weight: 500;
-}
-
-@font-face {
-  font-family: 'utopian-inter';
-  src: url('../fonts/Inter-MediumItalic.woff2') format('woff');
-  font-weight: 500;
-  font-style: italic;
-}
-
-@font-face {
-  font-family: 'utopian-inter';
-  src: url('../fonts/Inter-SemiBold.woff2') format('woff');
-  font-weight: 600;
-}
-
-@font-face {
-  font-family: 'utopian-inter';
-  src: url('../fonts/Inter-SemiBoldItalic.woff2') format('woff');
-  font-weight: 600;
-  font-style: italic;
-}
-
-* {
-  scrollbar-width: none;
-}
-
 body {
   margin: 0;
-  background-color: transparent;
-  font-family: 'utopian-inter', 'sans-serif';
-  font-size: 11px !important;
-  font-feature-settings: 'cv01', 'tnum', 'zero';
-  letter-spacing: 0.2px;
-  line-height: 17px;
-  overflow: hidden;
-  outline: none;
-  overscroll-behavior-x: contain;
-}
-
-a {
-  text-decoration: none;
-  color: #007aff;
 }
```

Comment:
This was required as the Hydrogen project includes the `body` tag, which by default has a margin. We've always been setting the margin to zero and accidentally relying on the editor CSS bleeding